webapi/src/main/twirl/queries/sparql/v2/getResourcePropertiesAndValuesGraphDB.scala.txt

```diff
@@ -101,6 +105,12 @@ CONSTRUCT {
                         ?valueObject knora-base:valueHasStandoff ?standoffNode .
                         ?standoffNode ?standoffProperty ?standoffValue .
                     }
+                } UNION {
+                     GRAPH <http://www.ontotext.com/explicit> {
+                        ?valueObject a knora-base:ListValue .
+                        ?valueObject knora-base:valueHasListNode  ?listNode .
+                        ?listNode knora-base:listNodeName ?listNodeName .
```

Comment:
@benjamingeer instead of `knora-base:listNodeName`, the `rdfs:label` should be queried.

Shall I change that?

Reply:
Yes I think so, keeping in mind that multiple languages may be returned, so `SparqlExtendedConstructRequest` will have to be used for the request, and `SparqlExtendedConstructResponse` will be used for the response.

Reply:
Ok, in that case we would have to use `SparqlExtendedConstructRequest` and `SparqlExtendedConstructResponse` in `ResourcesResponderV2` inside the method `getResources`. Maybe we can look at this tomorrow.